ansible/roles/google_cloud/tasks/main.yml

```diff
@@ -2,5 +2,5 @@
 # - name: Create functional account
   # include: role.yml
 
-- name: Create cloud trigger
-  include: function.yml
+# - name: Create cloud trigger
```

Comment:
Should this be removed?

Reply:
The other open branch will put it anyhow, leave it for later.